src/components/ProjectDetailCard/index.tsx

```diff
@@ -0,0 +1,156 @@
+                        <div className="flex flex-col">
+                            <div className="text-[24px] font-bold text-neutrals8">700,000</div>
+                            <div className="text-[16px] font-bold text-neutrals8">RSS3</div>
+                        </div>
+                        <Add width={24} height={24} />
+                        <Ton width={48} height={48} />
+                        <div className="flex flex-col">
+                            <div className="text-[24px] font-bold text-neutrals8">20,000</div>
+                            <div className="text-[16px] font-bold text-neutrals8">TON</div>
+                        </div>
+                    </div>
+                    <div className="text-[14px] font-bold text-neutrals6">Total Rewards</div>
+                </div>
+                <div
+                    className="flex h-[152px] w-full flex-col items-center gap-[24px] rounded-[16px] border-[1px] border-[white/0.07] bg-[white/0.1] p-[24px] lg:max-w-[350px]"
+                    style={{
+                        boxShadow:
+                            '0px 5px 10px 0px rgba(0, 0, 0, 0.05), 0px 15px 30px 0px rgba(0, 0, 0, 0.05), 0px 20px 40px 0px rgba(0, 0, 0, 0.05)',
+                    }}
+                >
+                    <div className="text-[32px] font-semibold text-neutrals8">
+                        {' '}
+                        {Math.floor(apyInfo?.data?.apy) || 0}%
+                    </div>
+                    <div className="text-[14px] font-bold text-neutrals6">APY</div>
+                </div>
+                <div
+                    className="flex h-[152px] w-full flex-col items-center gap-[24px] rounded-[16px] border-[1px] border-[white/0.07] bg-[white/0.1] p-[24px] lg:max-w-[350px]"
+                    style={{
+                        boxShadow:
+                            '0px 5px 10px 0px rgba(0, 0, 0, 0.05), 0px 15px 30px 0px rgba(0, 0, 0, 0.05), 0px 20px 40px 0px rgba(0, 0, 0, 0.05)',
+                    }}
+                >
+                    <div className="flex items-center gap-[4px] text-[32px] font-semibold text-neutrals8">
+                        {' '}
+                        {Math.floor(apyInfo?.data?.amount) || 0}
+                        <Mask width={36} height={36} />
+                    </div>
+                    <div className="text-[14px] font-bold text-neutrals6">Total MASK Staked</div>
+                </div>
+            </div>
+            <div className="mt-[24px] font-bold text-neutrals9">
+                This is the first phase of the MASK staking event. You will be able to retrieve your staked Mask tokens
+                and reward tokens only after the event concludes.
+            </div>
+            <MainButton
+                className="mt-[24px] w-[255px] !text-neutrals1"
+                onClick={() => {
+                    setIsOpen(true);
+                }}
+                style={{
+                    background: '#05050A',
```

Comment:
Use the tailwind way.

Reply:
tailwind will be overwritten by linear gradient background